packages/rest-crud/src/crud-rest.controller.ts

```diff
@@ -208,7 +208,7 @@ export function defineCrudRestController<
     async create(
       @body(modelCtor, {
         title: `New${modelName}`,
-        exclude: modelCtor.getIdProperties() as (keyof T)[],
+        exclude: modelCtor.getExcludeProperties() as (keyof T)[],
```

Comment:
Have you tested this with ids which are not auto increment and is generated as uuid in DB directly. In this case id is a string. So, I believe your change will cause a failure there. May be if you add a test case for such case and try it out, you might get to know.

Reply:
Hi, @samarpanB. I have tested the changes with Mongodb and it's working fine. This change will not have any effect if the id is a string as this is _only_ allowing the id to be passed while a post request if the id has been generated is set to false.

All the tests in rest-crud are based on memory-based datasource. We'll need to add MySQL/MongoDB-based data sources to add tests for the mentioned scenario.